app/controllers/ViewAttachmentController.scala

```diff
@@ -47,7 +47,7 @@ class ViewAttachmentController @Inject() (
             url     <- OptionT.fromOption[Future](fileSubmitted.url)
             content <- OptionT(fileService.downloadFile(url))
```

Comment:
You could potentially add a `mimeType` generator here too to return 404 rather than an empty mime type? I don't think this one makes a huge difference either way 😛